oracledb-mixin/dashboards/overview.libsonnet

```diff
@@ -5,7 +5,7 @@ local dashboardUid = 'oracledb-overview';
 
 local prometheus = grafana.prometheus;
 local promDatasourceName = 'prometheus_datasource';
-local matcher = 'job=~"$job", instance=~"$instance"';
+local getMatcher(cfg) = '%(tensorflowSelector)s, instance=~"$instance"' % cfg;
```

Comment:
tensorflowSelector to `oracledbSelector`. Also make sure that you run this in a local cluster with `enableMultiCluster` set to both true and false and see changes (new selector & panel queries have extra `cluster` filter)